[Controls] Fix control styles and invalid selection icon for Borealis

cqliu1 opened the pull request:
## Summary

Closes https://github.com/elastic/kibana/issues/210614.
Closes https://github.com/elastic/kibana/issues/210615.
Closes https://github.com/elastic/kibana/issues/210616.

This is a follow up to the Borealis updates to fix controls colors/styles. This also fixes the text color of the range slider number inputs to match the invalid state.

#### Light mode
![Screenshot 2025-03-10 at 5 54 28 PM](https://github.com/user-attachments/assets/1f3cfe7c-0a0f-4fed-9486-3aaee1b33fb1)

#### Dark mode
![Screenshot 2025-03-10 at 5 33 48 PM](https://github.com/user-attachments/assets/dda36740-93fd-4adc-bffe-c12aa4be3b4c)

#### Options List control with selection that results in no data

![Screenshot 2025-03-10 at 5 49 05 PM](https://github.com/user-attachments/assets/b8b6310b-c600-4703-af28-ee60d16339e0)

#### Range Slider control with valid selection that results in no data
![Screenshot 2025-03-10 at 5 49 01 PM](https://github.com/user-attachments/assets/afbf971b-95df-414f-8b99-2e3e0822fc73)

#### Range Slider control with invalid selection
![Screenshot 2025-03-10 at 5 48 48 PM](https://github.com/user-attachments/assets/dae00db3-8c94-47e5-a706-b43fa9e0787e)




### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [ ] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

### Identify risks

Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.

- [ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
- [ ] ...





<!--ONMERGE {"backportTargets":["8.18","8.x","9.0"]} ONMERGE-->

cqliu1 committed:
Remove transparent background on range slider

Remove more background styles

Fix invalid selection colors

Remove unused prop

Adjust warning colors

Add dep to hook

[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

Fix typo

[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

Switch to textWarning

Change back to EuiToken

Remove warning icon and invalid selection styles from options list control popover

[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

Revert dashboard unsaved changes badge color change

Remove unused imports

Fix typo

[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

Restore warning icon in options list popover

elasticmachine commented:
Pinging @elastic/kibana-presentation (Team:Presentation)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/282528)
* Commit: 477b641a6d4b9d772a8b04f5cf8bec3da88a35b1

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/282528#019582c5-9a69-4713-8c8e-a463ec83e2c3)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/282528","number":282528,"commit":"477b641a6d4b9d772a8b04f5cf8bec3da88a35b1"}],"number":282528}
buildkite-pr-comment-->

ThomThomson commented on src/platform/plugins/shared/controls/public/controls/data_controls/range_slider/get_range_slider_control_factory.tsx:
```diff
@@ -243,7 +243,7 @@ export const getRangesliderControlFactory = (): DataControlFactory<
               controlPanelClassName={controlPanelClassName}
               fieldFormatter={fieldFormatter}
-              isInvalid={selectionHasNotResults}
+              isInvalid={Boolean(value) && selectionHasNotResults}
```
Why the addition of `Boolean(value)` here? Also there's a typo in this line that's been around for a while looks like.

cqliu1 replied:
I opted to add this check because if the user hasn't selected any values in the range slider, it seems a little confusing to show the "your selection results in no data message" when the preceding controls already return no data.

Also I didn't even notice the typo haha. I'll fix it since I'm in here.

ThomThomson commented on src/platform/plugins/shared/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx:
```diff
@@ -271,7 +271,7 @@ export function InternalDashboardTopNav({
         badgeText: unsavedChangesBadgeStrings.getUnsavedChangedBadgeText(),
         title: '',
-        color: '#F6E58D',
+        color: 'warning',
```
Nice fix!

Heenawter replied:
FYI this change was made in https://github.com/elastic/kibana/pull/189998 and it unified the "unsaved changes" badge color with Discover, as well. If we make this change, we will be different from Discover:

| Discover | Dashboard in this PR |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/78c8c71f-9707-40db-b5bc-8a8ff6a01ab6) | ![image](https://github.com/user-attachments/assets/ae206a17-e82d-41aa-a364-1f1aafb8280e) | 

Do we want this?

ThomThomson reviewed (approved):
Design changes LGTM! code only review

andreadelrio reviewed (changes requested):
![image](https://github.com/user-attachments/assets/7d70adb7-8269-4eb6-bf14-2845323021b4)

We have an issue with accessibility and this yellow on white combination in light mode.
I suggest replacing `color={euiTheme.colors.warning}` with `color={euiTheme.colors. textSubdued}`.

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/283031)
* Commit: 6059073490879610ce0cb9cc5d2df46feb2ae818

### Failed CI Steps
* [FTR Configs #121](https://buildkite.com/elastic/kibana-pull-request/builds/283031#0195876e-6fe7-498e-b02d-552bc15c1282)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/283031#0195876e-6fe7-498e-b02d-552bc15c1282) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/283031/jobs/0195876e-6fe7-498e-b02d-552bc15c1282/artifacts/01958794-67db-4b9e-9407-524e4ef310f8) FTR Configs #<span></span>121 / @<span></span>ess @<span></span>serverless @<span></span>serverlessQA SecuritySolution Timeline Timeline @<span></span>skipInServerless resolve timeline "before all" hook for "should return outcome exactMatch when the id is unchanged"

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/c835902ff9a277f306a5d6ab46bf1924088fd2dd) | [after](https://github.com/elastic/kibana/commit/6059073490879610ce0cb9cc5d2df46feb2ae818) | diff |
| --- | --- | --- | --- |
| `controls` | 414.7KB | 414.5KB | -270.0B |



### History
* :broken_heart: [Build #282528](https://buildkite.com/elastic/kibana-pull-request/builds/282528) failed 477b641a6d4b9d772a8b04f5cf8bec3da88a35b1

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283031","number":283031,"commit":"6059073490879610ce0cb9cc5d2df46feb2ae818"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/282528","number":282528,"commit":"477b641a6d4b9d772a8b04f5cf8bec3da88a35b1"}],"number":283031}
buildkite-pr-comment-->

cqliu1 commented:
@andreadelrio What if we used `textWarning` instead of `textSubdued`? It'd match the text color on invalid controls selections.
![Screenshot 2025-03-13 at 12 27 16 PM](https://github.com/user-attachments/assets/87963597-d47d-4a00-a2e9-ccd81c1723f5)
 
![Screenshot 2025-03-13 at 12 27 12 PM](https://github.com/user-attachments/assets/c85cf782-941b-418d-bd3c-77dec4543972)

It looks better than `textSubdued`.
![Screenshot 2025-03-13 at 12 30 57 PM](https://github.com/user-attachments/assets/b8c0e7d7-ef01-42b8-bb08-3bacbce28e67)

![Screenshot 2025-03-13 at 12 30 20 PM](https://github.com/user-attachments/assets/ba7ac814-1c6b-447f-8880-fca583261f56)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/283905)
* Commit: 58e3504778270da4874d70a678a0a09dc528feee

### Failed CI Steps
* [Explore - Security Solution Cypress Tests #2](https://buildkite.com/elastic/kibana-pull-request/builds/283905#019590fc-d83a-4cd2-a1ca-4bf12d7bf68e)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/283905#019590fc-d83a-4cd2-a1ca-4bf12d7bf68e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/283905/jobs/019590fc-d83a-4cd2-a1ca-4bf12d7bf68e/artifacts/01959133-e4e9-488b-bbc6-60c36f449eb5) Explore - Security Solution Cypress Tests #<span></span>2 / Inspect Explore pages inspect Hosts page inspect Hosts page

### History
* :yellow_heart: [Build #283031](https://buildkite.com/elastic/kibana-pull-request/builds/283031) was flaky 6059073490879610ce0cb9cc5d2df46feb2ae818
* :broken_heart: [Build #282528](https://buildkite.com/elastic/kibana-pull-request/builds/282528) failed 477b641a6d4b9d772a8b04f5cf8bec3da88a35b1

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283905","number":283905,"commit":"58e3504778270da4874d70a678a0a09dc528feee"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283031","number":283031,"commit":"6059073490879610ce0cb9cc5d2df46feb2ae818"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/282528","number":282528,"commit":"477b641a6d4b9d772a8b04f5cf8bec3da88a35b1"}],"number":283905}
buildkite-pr-comment-->

andreadelrio commented:
> @andreadelrio What if we used `textWarning` instead of `textSubdued`? It'd match the text color on invalid controls selections. ![Screenshot 2025-03-13 at 12 27 16 PM](https://private-user-images.githubusercontent.com/1697105/422509161-87963597-d47d-4a00-a2e9-ccd81c1723f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE4OTg1ODAsIm5iZiI6MTc0MTg5ODI4MCwicGF0aCI6Ii8xNjk3MTA1LzQyMjUwOTE2MS04Nzk2MzU5Ny1kNDdkLTRhMDAtYTJlOS1jY2Q4MWMxNzIzZjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTNUMjAzODAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzM4ZWZkOGVkOGE1ZTVkMzE3NWVlNWJjNDViNjg2N2JkOTA0NTNjYTE5ZTZmMzgwNjNiZTY5YzNhOWJlYTNlNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.3SCMAh81mnwBc8nXFMZfX76tlyS7k6Me-wLNt_EVDdQ)
> 
> ![Screenshot 2025-03-13 at 12 27 12 PM](https://private-user-images.githubusercontent.com/1697105/422509159-c85cf782-941b-418d-bd3c-77dec4543972.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE4OTg1ODAsIm5iZiI6MTc0MTg5ODI4MCwicGF0aCI6Ii8xNjk3MTA1LzQyMjUwOTE1OS1jODVjZjc4Mi05NDFiLTQxOGQtYmQzYy03N2RlYzQ1NDM5NzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTNUMjAzODAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2U3MDdmYjM4OWQ2ODUyZjI5Mjg5YWUwZDFlNGVmYzFiYjQ5YWEwNzgxYjQ2ZGY0OGU1OTI5YTdlZTI4YzYzZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.vKhmtfDHmj0ZZkcl6HFi2pQyxZ6xUQdUqLtva0NBp3g)
> 
> It looks better than `textSubdued`. ![Screenshot 2025-03-13 at 12 30 57 PM](https://private-user-images.githubusercontent.com/1697105/422509322-b8c0e7d7-ef01-42b8-bb08-3bacbce28e67.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE4OTg1ODAsIm5iZiI6MTc0MTg5ODI4MCwicGF0aCI6Ii8xNjk3MTA1LzQyMjUwOTMyMi1iOGMwZTdkNy1lZjAxLTQyYjgtYmIwOC0zYmFjYmNlMjhlNjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTNUMjAzODAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzkwMjE4MDBjM2Y4YWVkMzE0NWJkZGM5NGEzMGM0ZDYxNWM0YzU0Nzg3YmRmYWM5ZGIxYzQ0ZDhjZTY3ZGI0MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.7_-fFUG46COfuParRg9ZsWtjTb8ko4t_2l7YW-1SCWo)
> 
> ![Screenshot 2025-03-13 at 12 30 20 PM](https://private-user-images.githubusercontent.com/1697105/422509316-ba7ac814-1c6b-447f-8880-fca583261f56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE4OTg1ODAsIm5iZiI6MTc0MTg5ODI4MCwicGF0aCI6Ii8xNjk3MTA1LzQyMjUwOTMxNi1iYTdhYzgxNC0xYzZiLTQ0N2YtODg4MC1mY2E1ODMyNjFmNTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTNUMjAzODAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODdmZTYxN2MxNjgzMWI2YTkyNjlhYTQ3ODc0ZWFlMzg5N2YzZjk2NTlmYWU0MDBkYzU1ODZkODcyMWZjOWUyNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ._gl820TFXLeFVObnh8PPBDScDo9AjAX1Q86S5wPV3g8)

@cqliu1 Upon taking a closer look I think that for the warning in the input we should continue using **EuiToken** and simply pass `euiColorVis9` instead.

<img width="601" alt="image" src="https://github.com/user-attachments/assets/071a2218-725f-4ed1-8ffc-5901af24e9dc" />

For the popover, I'm not sure we need icons or text in yellow to be honest. I'd vote for simplifying and using the same dark gray when use in the rest of the popover.

cqliu1 commented:
@andreadelrio I switched back to the `EuiToken` component using the `warning` icon with the color `euiVisColor9` for both the options list and range slider controls.

![Screenshot 2025-03-17 at 2 27 16 PM](https://github.com/user-attachments/assets/0fc4ce56-7cc6-498a-a815-db41f8731947)

![Screenshot 2025-03-17 at 2 26 58 PM](https://github.com/user-attachments/assets/7ec986e8-317d-4186-a1e8-d41a5f1e0557)

I also removed the warning icon and warning text color from the invalid selections from the options list control popover.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/284780)
* Commit: 5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/284780#0195a605-0741-4939-9340-5dc21d2a5684)

### History
* :broken_heart: [Build #283905](https://buildkite.com/elastic/kibana-pull-request/builds/283905) failed 58e3504778270da4874d70a678a0a09dc528feee
* :yellow_heart: [Build #283031](https://buildkite.com/elastic/kibana-pull-request/builds/283031) was flaky 6059073490879610ce0cb9cc5d2df46feb2ae818
* :broken_heart: [Build #282528](https://buildkite.com/elastic/kibana-pull-request/builds/282528) failed 477b641a6d4b9d772a8b04f5cf8bec3da88a35b1

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284780","number":284780,"commit":"5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283905","number":283905,"commit":"58e3504778270da4874d70a678a0a09dc528feee"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283031","number":283031,"commit":"6059073490879610ce0cb9cc5d2df46feb2ae818"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/282528","number":282528,"commit":"477b641a6d4b9d772a8b04f5cf8bec3da88a35b1"}],"number":284780}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/285037)
* Commit: 51d0090ec77868b1e44c36c494a5fd0d0a1a19e6

### Failed CI Steps
* [Build Kibana Distribution](https://buildkite.com/elastic/kibana-pull-request/builds/285037#0195a9c6-2304-4ce8-8c40-314fbe95bc60)

### History
* :broken_heart: [Build #284785](https://buildkite.com/elastic/kibana-pull-request/builds/284785) failed 5e14082a01cbf57fda9279475a9864c1c82ecf62
* :broken_heart: [Build #284780](https://buildkite.com/elastic/kibana-pull-request/builds/284780) failed 5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3
* :broken_heart: [Build #283905](https://buildkite.com/elastic/kibana-pull-request/builds/283905) failed 58e3504778270da4874d70a678a0a09dc528feee
* :yellow_heart: [Build #283031](https://buildkite.com/elastic/kibana-pull-request/builds/283031) was flaky 6059073490879610ce0cb9cc5d2df46feb2ae818
* :broken_heart: [Build #282528](https://buildkite.com/elastic/kibana-pull-request/builds/282528) failed 477b641a6d4b9d772a8b04f5cf8bec3da88a35b1

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285037","number":285037,"commit":"51d0090ec77868b1e44c36c494a5fd0d0a1a19e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284785","number":284785,"commit":"5e14082a01cbf57fda9279475a9864c1c82ecf62"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284780","number":284780,"commit":"5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283905","number":283905,"commit":"58e3504778270da4874d70a678a0a09dc528feee"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283031","number":283031,"commit":"6059073490879610ce0cb9cc5d2df46feb2ae818"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/282528","number":282528,"commit":"477b641a6d4b9d772a8b04f5cf8bec3da88a35b1"}],"number":285037}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/285110)
* Commit: f95ee52a95e61f6a2015473a7a4b9b2ece40d637

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/285110#0195aa52-ac3c-40dd-af09-b2990b380b96)

### History
* :broken_heart: [Build #285037](https://buildkite.com/elastic/kibana-pull-request/builds/285037) failed 51d0090ec77868b1e44c36c494a5fd0d0a1a19e6
* :broken_heart: [Build #284785](https://buildkite.com/elastic/kibana-pull-request/builds/284785) failed 5e14082a01cbf57fda9279475a9864c1c82ecf62
* :broken_heart: [Build #284780](https://buildkite.com/elastic/kibana-pull-request/builds/284780) failed 5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3
* :broken_heart: [Build #283905](https://buildkite.com/elastic/kibana-pull-request/builds/283905) failed 58e3504778270da4874d70a678a0a09dc528feee
* :yellow_heart: [Build #283031](https://buildkite.com/elastic/kibana-pull-request/builds/283031) was flaky 6059073490879610ce0cb9cc5d2df46feb2ae818

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285110","number":285110,"commit":"f95ee52a95e61f6a2015473a7a4b9b2ece40d637"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285037","number":285037,"commit":"51d0090ec77868b1e44c36c494a5fd0d0a1a19e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284785","number":284785,"commit":"5e14082a01cbf57fda9279475a9864c1c82ecf62"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284780","number":284780,"commit":"5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283905","number":283905,"commit":"58e3504778270da4874d70a678a0a09dc528feee"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283031","number":283031,"commit":"6059073490879610ce0cb9cc5d2df46feb2ae818"}],"number":285110}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/285131)
* Commit: 23c6f614eae55e678940db079e420c389fae4e96

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/6613e27804e55f4d82026f218e84e991c7e9151e) | [after](https://github.com/elastic/kibana/commit/23c6f614eae55e678940db079e420c389fae4e96) | diff |
| --- | --- | --- | --- |
| `controls` | 414.7KB | 414.2KB | -595.0B |



### History
* :broken_heart: [Build #285110](https://buildkite.com/elastic/kibana-pull-request/builds/285110) failed f95ee52a95e61f6a2015473a7a4b9b2ece40d637
* :broken_heart: [Build #285037](https://buildkite.com/elastic/kibana-pull-request/builds/285037) failed 51d0090ec77868b1e44c36c494a5fd0d0a1a19e6
* :broken_heart: [Build #284785](https://buildkite.com/elastic/kibana-pull-request/builds/284785) failed 5e14082a01cbf57fda9279475a9864c1c82ecf62
* :broken_heart: [Build #284780](https://buildkite.com/elastic/kibana-pull-request/builds/284780) failed 5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3
* :broken_heart: [Build #283905](https://buildkite.com/elastic/kibana-pull-request/builds/283905) failed 58e3504778270da4874d70a678a0a09dc528feee
* :yellow_heart: [Build #283031](https://buildkite.com/elastic/kibana-pull-request/builds/283031) was flaky 6059073490879610ce0cb9cc5d2df46feb2ae818

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285131","number":285131,"commit":"23c6f614eae55e678940db079e420c389fae4e96"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285110","number":285110,"commit":"f95ee52a95e61f6a2015473a7a4b9b2ece40d637"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285037","number":285037,"commit":"51d0090ec77868b1e44c36c494a5fd0d0a1a19e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284785","number":284785,"commit":"5e14082a01cbf57fda9279475a9864c1c82ecf62"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284780","number":284780,"commit":"5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/283905","number":283905,"commit":"58e3504778270da4874d70a678a0a09dc528feee"}],"number":285131}
buildkite-pr-comment-->

cqliu1 commented:
Per @andreadelrio request, I restored the warning icon in the options list popover and made it match the color of the text instead of using the `warning` color.

cqliu1 committed:
Change to warning icon

andreadelrio reviewed (approved):
LGTM. Thanks!

kibanamachine commented:
Starting backport for target branches: 8.18, 8.x, 9.0

https://github.com/elastic/kibana/actions/runs/14183925523

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288809)
* Commit: 77488e181ab50bdebd5c6b67d395122d6f1bf7c7

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/63575a8320a7e021df79727ad66ed4cf01dd1e5f) | [after](https://github.com/elastic/kibana/commit/77488e181ab50bdebd5c6b67d395122d6f1bf7c7) | diff |
| --- | --- | --- | --- |
| `controls` | 420.9KB | 420.5KB | -458.0B |



### History
* :green_heart: [Build #285131](https://buildkite.com/elastic/kibana-pull-request/builds/285131) succeeded 23c6f614eae55e678940db079e420c389fae4e96
* :broken_heart: [Build #285110](https://buildkite.com/elastic/kibana-pull-request/builds/285110) failed f95ee52a95e61f6a2015473a7a4b9b2ece40d637
* :broken_heart: [Build #285037](https://buildkite.com/elastic/kibana-pull-request/builds/285037) failed 51d0090ec77868b1e44c36c494a5fd0d0a1a19e6
* :broken_heart: [Build #284785](https://buildkite.com/elastic/kibana-pull-request/builds/284785) failed 5e14082a01cbf57fda9279475a9864c1c82ecf62
* :broken_heart: [Build #284780](https://buildkite.com/elastic/kibana-pull-request/builds/284780) failed 5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3
* :broken_heart: [Build #283905](https://buildkite.com/elastic/kibana-pull-request/builds/283905) failed 58e3504778270da4874d70a678a0a09dc528feee

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288809","number":288809,"commit":"77488e181ab50bdebd5c6b67d395122d6f1bf7c7"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285131","number":285131,"commit":"23c6f614eae55e678940db079e420c389fae4e96"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285110","number":285110,"commit":"f95ee52a95e61f6a2015473a7a4b9b2ece40d637"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/285037","number":285037,"commit":"51d0090ec77868b1e44c36c494a5fd0d0a1a19e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284785","number":284785,"commit":"5e14082a01cbf57fda9279475a9864c1c82ecf62"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/284780","number":284780,"commit":"5176e05b6cb4adb5f5c51c89d5b0da85c7040ad3"}],"number":288809}
buildkite-pr-comment-->

kibanamachine commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.18|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/216567">](https://github.com/elastic/kibana/pull/216567)|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/216568">](https://github.com/elastic/kibana/pull/216568)|
|✅|9.0|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/216569">](https://github.com/elastic/kibana/pull/216569)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->
